src/rich-log/options.hh

```diff
@@ -4,30 +4,40 @@
 
 namespace rlog
 {
-// not enum class so that LOG(info) works
-enum channel
-{
-    info,
-    warning,
-    error,
-    debug
-};
-
 struct prefix
 {
-    explicit prefix(char const* v) : value(v) {}
+    explicit constexpr prefix(char const* v) : value(v) {}
 
     char const* value;
 };
 
 struct sep
 {
-    explicit sep(char const* v) : value(v) {}
+    explicit constexpr sep(char const* v) : value(v) {}
 
     char const* value;
 };
 
 static constexpr struct no_sep_t
 {
 } no_sep;
+
+static constexpr struct err_out_t
```

Comment:
hm I don't think this should be part of the API.
Once we get proper log writers, "err out" might not even be a valid concept anymore.